workers/unity/Assets/Config/DeploymentLauncherConfig.asset

```diff
@@ -31,6 +31,7 @@ MonoBehaviour:
     SimulatedPlayerDeploymentConfigs:
     - Name: my_deployment_sim1
       SnapshotPath: 
+      SnapshotPath: snapshots/cloud.snapshot
```

Comment:
Is github drunk, or is this property saved twice?

Reply:
😕

Reply:
I'm very confused, I don't even have this change in my local branch

Reply:
Managed to resolve it